config.md

```diff
@@ -128,27 +131,25 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001's `environ`][ieee-1003.1-2001-xbd-c8.1].
 * **`args`** (array of strings, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execvp`'s *argv*][ieee-1003.1-2001-xsh-exec].
   This specification extends the IEEE standard in that at least one entry is REQUIRED, and that entry is used with the same semantics as `execvp`'s *file*.
-
-For Linux-based systems the process structure supports the following process specific fields:
-
-* **`capabilities`** (array of strings, OPTIONAL) capabilities is an array that specifies Linux capabilities that can be provided to the process inside the container.
-  Valid values are the strings for capabilities defined in [the man page](http://man7.org/linux/man-pages/man7/capabilities.7.html).
+* **`capabilities`** (array of strings, OPTIONAL) is an array that specifies the set of capabilities of the process(es) inside the container. Valid values are platform-specific. For example, valid values for Linux are defined in the [CAPABILITIES(7)](http://man7.org/linux/man-pages/man7/capabilities.7.html) man page.
```

Comment:
> then the runtime errors out. end-users investigate. complain to the runtime developers or write a fix. That kind-of linting and validation needs to be secondary to this spec.

So what kind of `CAP_*` values should the integration test suite use?  Currently it [sets][1] [these][2], and fails runtimes which [die in `create`][3] (currently `start`, because runtime-tools hasn't been ported around #384, opencontainers/runtime-tools#285) or [run without a matching set][4].  Will we update that to say that any runtime is compliant as long as it:

* Dies in `create` or
* Succeeds in `create` and has a container environment matching the configuration

And then test all combinations (or some sampling) of `CAP_*` values regardless of platform support?

[1]: https://github.com/opencontainers/runtime-tools/blob/2d92f6557e64d4f9a0e799a75fdf153cec13dffa/test_runtime.sh#L80
[2]: https://github.com/opencontainers/runtime-tools/blob/2d92f6557e64d4f9a0e799a75fdf153cec13dffa/generate/generate.go#L57-L70
[3]: https://github.com/opencontainers/runtime-tools/blob/2d92f6557e64d4f9a0e799a75fdf153cec13dffa/test_runtime.sh#L82-L85
[4]: https://github.com/opencontainers/runtime-tools/blob/2d92f6557e64d4f9a0e799a75fdf153cec13dffa/cmd/runtimetest/main.go#L175-L194